ci-operator/jobs/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-hyperconverged-cluster-operator-master-presubmits.yaml

```diff
@@ -0,0 +1,116 @@
+presubmits:
+  kubevirt/hyperconverged-cluster-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - hyperconverged-cluster-operator-master
+    context: ci/prow/hco-e2e
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    name: pull-ci-kubevirt-hyperconverged-cluster-operator-hyperconverged-cluster-operator-master-hco-e2e
+    rerun_command: /test hco-e2e
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/hco-e2e-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=hco-e2e
+        - --template=/usr/local/hco-e2e
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: kubevirt-hyperconverged-cluster-operator-hyperconverged-cluster-operator-master.yaml
+              name: ci-operator-misc-configs
+        - name: JOB_NAME_SAFE
+          value: hco-e2e
+        - name: TEST_COMMAND
```

Comment:
@rthallisey FYI. Some work on the makefile needed but would this command execution make sense at the end?

Reply:
I don't see a problem with it